engine/flink/test-util/src/main/scala/pl/touk/nussknacker/engine/flink/test/FlinkMiniClusterHolder.scala

```diff
@@ -25,12 +28,18 @@ trait FlinkMiniClusterHolder {
 
   def stop(): Unit
 
+  def cancelJob(jobID: JobID): Unit
```

Comment:
Let's add `private[test]` for those four methods - let's developers use execution environment instead

Reply:
Done